contributions/api/utils/adminutils.py

```diff
@@ -54,54 +54,100 @@ def check_if_reviewer(login_id):
 
     return False
 
-def send_email(mail_to, subject, message):
+def establish_smtp_connection():
+    """
+    Method to establish SMTP connection
+    Args: None
+    Returns:
+        connection (obj): SMTP object
+        smtp_error (str): Error string
+        smtp_code (str): Error code
+    """
+    mail_from = cfg.SENDER_EMAIL
+    password = cfg.SENDER_EMAIL_PASSWORD
+    connection = smtplib.SMTP(host=cfg.SMTP_HOST, port=cfg.SMTP_PORT)
+    try:
+        connection.ehlo()
+    except smtplib.SMTPHeloError as ex:
+        return False, ex.smtp_error, ex.smtp_code
```

Comment:
Based on the return type description at beginning of this method, the first value here may need to be updated to None. Similarly in other return statements in this method too. What do you think?

Reply:
updated to None